src/ngx_lua_resty_lmdb_module.c

```diff
@@ -363,6 +375,147 @@ ngx_lua_resty_lmdb_verify_file_status(ngx_cycle_t *cycle,
 }
 
 
+static ngx_int_t
+ngx_lua_resty_lmdb_validate(ngx_cycle_t *cycle,
+                            ngx_lua_resty_lmdb_conf_t *lcf)
+{
+    int                        rc;
+    MDB_dbi                    dbi;
+    MDB_val                    key;
+    MDB_val                    value;
+    MDB_txn                   *txn = NULL;
+
+    ngx_str_t                  validation_key =
+                                    ngx_string(NGX_LUA_RESTY_LMDB_VALIDATION_KEY);
+
+    /* check tag value in lmdb */
+
+    if (lcf->validation_tag.data == NULL) {
+        ngx_log_debug0(NGX_LOG_DEBUG_CORE, cycle->log, 0,
+                       "LMDB validation disabled");
```

Comment:
Instead of "LMDB validation disabled", we should print debug log "LMDB validation enabled, using validation tag: \"%V\"" when this feature is enabled.

Reply:
done.